### PR TITLE
workflow/s3-cloudfront

### PR DESCRIPTION
**Pull request by omartehsin1**
# Summary | Résumé

Created a new workflow file which will push to the new EN and FR S3 buckets and will deploy and invalidate the new cloudfront distributions.

### CODE REVIEW

**Comment by omartehsin1** [comment]
[Terraform Github worklow](https://trello.com/c/l9GyoxR6/476-terraform-github-worklow)

File: .github/workflows/build-push.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```
Comment (patheard) [inline .github/workflows/build-push.yml:13]: Could you bump this to v3:
```suggestion
    - uses: actions/checkout@v3
```

File: .github/workflows/build-push.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    - name: Load git submodules
+      run: git submodule sync && git submodule update --init
+
+    - name: Install hugo
+      run: |
+        wget https://github.com/gohugoio/hugo/releases/download/v0.113.0/hugo_extended_0.113.0_Linux-64bit.tar.gz \
+        && tar -xvf hugo_extended_0.113.0_Linux-64bit.tar.gz hugo \
+        && mv  hugo /usr/local/bin \
+        && rm hugo_extended_0.113.0_Linux-64bit.tar.gz
+    - name: Build english and french sites
```
Comment (patheard) [inline .github/workflows/build-push.yml:24]: nit: could you add a newline:
```suggestion

    - name: Build english and french sites
```

File: .github/workflows/build-push.yml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      run: |
+        hugo --ignoreCache -d ~/cds-website-dist
+        chmod -R 755 ~/cds-website-dist
+
+    - name: Cache build
+      uses: actions/cache@v2
```
Comment (patheard) [inline .github/workflows/build-push.yml:30]: Could you bump this to v3 as well please:
```suggestion
      uses: actions/cache@v3
```

**Review by patheard — APPROVED** [review]

**Pull request merged**